user_guide_src/source/database/configuration.rst

```diff
@@ -174,7 +174,7 @@ Explanation of Values:
 **compress**    Whether or not to use client compression (``MySQLi`` only).
 **strictOn**    true/false (boolean) - Whether to force "Strict Mode" connections, good for ensuring strict SQL
                 while developing an application (``MySQLi`` only).
-**port**        The database port number.
+**port**        The database port number - Empty for default port (or dynamic port with ``SQLSRV``)
```

Comment:
Do you mean an empty string?

Reply:
yes - empty string is the only value that works for all driver 
( sqlsrv and postgre doesnt use empty function for validation )